core/domain/subtopic_page_domain_test.py

```diff
@@ -234,50 +229,52 @@ def test_update_written_translations(self):
         self.assertEqual(
             self.subtopic_page.to_dict(), expected_subtopic_page_dict)
 
-    def test_create_subtopic_page_change(self):
-        subtopic_page_change_object = subtopic_page_domain.SubtopicPageChange({
+    def test_create_subtopic_page_change(self) -> None:
+        subtopic_page_change_object = subtopic_page_domain.SubtopicPageChange(
+            {
             'cmd': subtopic_page_domain.CMD_CREATE_NEW,
             'topic_id': self.topic_id,
             'subtopic_id': 'subtopic_id'
         })
 
         self.assertEqual(
-            subtopic_page_change_object.to_dict(), {
+            subtopic_page_change_object.to_dict(),
+            {
                 'cmd': subtopic_page_domain.CMD_CREATE_NEW,
                 'topic_id': self.topic_id,
                 'subtopic_id': 'subtopic_id'
             })
 
-    def test_validate_version_number(self):
-        self.subtopic_page.version = 'invalid_version'
-        with self.assertRaisesRegex(
+    def test_validate_version_number(self) -> None:
+        self.subtopic_page.version = 'invalid_version' # type: ignore[assignment]
```

Comment:
Why is this ignore needed?

Reply:
A string is initentionally passed here where int is expected for testing purposes, hence added the ignore comment.

Reply:
I think you can remove this test case and also in the `validate` you can remove the `isinstance` check.

Reply:
Done!

Reply:
ditto elsewhere

Reply:
Ya will do!